### DISCUSSION TITLE
Antivirus Exclusion List

### DISCUSSION THREAD

**Discussion by jmchenry81** [rustdesk/rustdesk#855] [General]
Hi Everyone, moving from my lab environment to test in production (Vipre Endpoint Security), cannot install the service from the client, get's triggered on cmd.exe.

I have one of the techs @ Vipre trying to work through this with me. Any suggestions other than pausing AV to install service? =]

log file attached
[vipre.log.rustdesk.txt](https://github.com/rustdesk/rustdesk/files/8968882/vipre.log.rustdesk.txt)
:

**Comment by jmchenry81** [comment]
once the service is installed, all works fine again. And works before as a single use client. It's only installing the service that triggers

**Comment by rustdesk** [comment]
Honestly, I do not know. Hope some one with such expierence share their ideas.
